contracts/AnchorVault.vy

```diff
@@ -581,199 +237,34 @@ def withdraw(
     To withdraw Terra-side bETH, you should firstly transfer the tokens to the Ethereum
     blockchain.
 
-    The call fails if `AnchorVault.can_deposit_or_withdraw()` returns false.
-
     The conversion rate from stETH to bETH should normally be 1 but might be different after
-    severe penalties inflicted on the Lido validators. You can obtain the current conversion
```

Comment:
The conversion rate should be fixed between stETH shares at the moment of the latest upgrade and bETH in existence (with additional concern of the refunded part), I believe